docs/index.md

```diff
@@ -4,6 +4,12 @@ Standard Operating Procedures (SOPs) bundle the documentation that describes the
 Within your lab or your institution, the goal is that experimenters can carry out new data collection according to the overarching study plans with minimal supervision.
 Beyond the doors of your lab or institution, SOPs are a critical tool to maximize the reproducibility of your research by others.
 
+??? thanks "Thanks to Dr. Jo Etzel and to McKenzie P. Hagen"
+
+    Thanks Jo, for seeding the ideas and demonstrating the importance of SOPs in our scientific endeavor and your excellent work in the dissemination of [the DMCC Project](https://mfr.osf.io/render?url=https://osf.io/6r9f8/?direct%26mode=render%26action=download%26mode=render) that includes SOPs we have reused as examples here.
+    Thanks McKenzie, for lending your experience as an RA at Stanford to fill some of the early drafts of the acquisition examples and checklists.
```

Comment:
@mckenziephagen and @jaetzel I have added this acknowledgment and only after pushing it I thought you may want me to remove it. If that is the case please let me know. If you want to see how it shows up rendered -> https://www.nipreps.org/sops-cookiecutter/.

Either way (whether you do or you don't want this to be published)- THANK YOU.